actions/checkForAndPlaceOrder.ts

```diff
@@ -86,29 +103,49 @@ const _checkForAndPlaceOrder: ActionFn = async (
         chainContext.provider
       );
 
-      const { deleteConditionalOrder, error } = await _processConditionalOrder(
+      const pollResult = await _processConditionalOrder(
```

Comment:
I find this variable name misleading.

When truthy it means it's an error, so you only care for this in case it errors.

What about calling it `error` instead?
Or something that makes it clearer

Reply:
You are right!